app/controllers/ops_controller/diagnostics.rb

```diff
@@ -152,17 +152,15 @@ def log_depot_edit
   def fetch_log
     assert_privileges("fetch_log")
     disable_client_cache
-    send_data(Vmdb::Loggers.contents($log, nil),
-              :filename => "evm.log")
+    send_data(fetch_journal_log(:syslog_identifier => "evm"), :filename => "evm.log")
```

Comment:
This might be cleaner as something like:

```suggestion
    send_data(fetch_log_contents(:service => "evm"))
```

Then in the fetch_log_contents something like:

```
def fetch_log_contents(service:)
  fetch_journal_log(:syslog_identifier => service), :filename => "#{service}.log"
end
```

Reply:
Yeah I'm playing with the args to this method now I think just passing in `fetch_journal_log("evm")` is the normal case but we should allow additional filter params if the caller wants to filter down more.